src/components/AddNewTodoModal.tsx

```diff
@@ -0,0 +1,93 @@
+import { useState, useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import { TextField, Modal, Box, Button, Typography } from '@mui/material';
+import { addTodo } from '../actions/actionCreators';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { LocalizationProvider, DateTimePicker } from '@mui/x-date-pickers';
+import { getNewTaskObject } from '../utils/getNewTaskObject';
+
+const style = {
+  position: 'absolute',
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-around',
+  width: '400px',
+  height: '400px',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  bgcolor: 'background.paper',
+  border: '2px solid #000',
+  boxShadow: 24,
+  p: 4,
+};
+
+const currentDate = new Date();
```

Comment:
Do you need this variable in the global scope?

Reply:
no, I don't)